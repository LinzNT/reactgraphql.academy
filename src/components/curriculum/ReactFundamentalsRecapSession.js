import React from 'react'
import Ul, { Li } from '../layout/Ul'
import Session from './Session'

const ReactFundamentalsRecapSession = ({ hideTitle }) => (
  <Session
    hideTitle={hideTitle}
    title="React Fundamentals recap, build a React app from scratch on your own to
    consolidate:"
  >
    <Ul>
      <Li>React</Li>
      <Li>React Router</Li>
      <Li>Data fetching</Li>
    </Ul>
  </Session>
)

export default ReactFundamentalsRecapSession
