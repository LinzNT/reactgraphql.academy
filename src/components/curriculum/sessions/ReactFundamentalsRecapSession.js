import React from 'react'
import Ul, { Li } from '../../layout/Ul'
import Session from './Session'

const ReactFundamentalsRecapSession = ({ title }) => (
  <Session title={title}>
    <Ul>
      <Li>React</Li>
      <Li>React Router</Li>
      <Li>Data fetching</Li>
    </Ul>
  </Session>
)

export default ReactFundamentalsRecapSession
