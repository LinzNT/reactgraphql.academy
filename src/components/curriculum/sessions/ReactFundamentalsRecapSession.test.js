import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import ReactFundamentalsRecapSession from './ReactFundamentalsRecapSession'

describe('<ReactFundamentalsRecapSession />', () => {
  it('should render component', () => {
    const wrapper = shallow(<ReactFundamentalsRecapSession />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
