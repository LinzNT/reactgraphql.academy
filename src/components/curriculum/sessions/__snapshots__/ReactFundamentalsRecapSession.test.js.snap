// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReactFundamentalsRecapSession /> should render component 1`] = `
<Session>
  <styled.ul>
    <styled.li>
      React
    </styled.li>
    <styled.li>
      React Router
    </styled.li>
    <styled.li>
      Data fetching
    </styled.li>
  </styled.ul>
</Session>
`;
